blocks/identity-block/components/HeadlinedSubmitForm/styles.scss

```diff
@@ -24,6 +24,7 @@
 
   .xpmedia-button {
     margin: map-get($spacers, 'md') 0;
+    width: 100%;
```

Comment:
updated to match form mocks based on my changes

Reply:
We shouldn't set the width here - the button should control this - if we add a prop `fullWidth` or similar the button should apply the 100% width - there is likely many other places we will need to set this if it's not part of the button component

Reply:
updated